src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -155,39 +142,68 @@ namespace DataHeatBalance {
     constexpr int DefaultMinNumberOfWarmupDays(1);  // Default minimum number of warmup days allowed
 
     // Parameters for Sky Radiance Distribution
-    constexpr int Isotropic(0);
-    constexpr int Anisotropic(1);
+    enum class SkyRadDistribution
+    {
+        Unassigned = -1,
+        Isotropic,
+        Anisotropic
+    };
 
     // Parameters for ZoneAirSolutionAlgo
-    constexpr int Use3rdOrder(0);
-    constexpr int UseAnalyticalSolution(1);
-    constexpr int UseEulerMethod(2);
+    enum class SolutionAlgo
+    {
+        Unassigned = -1,
+        ThirdOrder,
+        AnalyticalSolution,
+        EulerMethod
+    };
 
     // Parameter for MRT calculation type
-    constexpr int ZoneAveraged(1);
-    constexpr int SurfaceWeighted(2);
-    constexpr int AngleFactor(3);
+    enum class CalcMRT
+    {
+        Unassigned = -1,
+        ZoneAveraged,
+        SurfaceWeighted,
+        AngleFactor
+    };
 
     // Parameters for Ventilation
-    constexpr int NaturalVentilation(0);
-    constexpr int IntakeVentilation(1);
-    constexpr int ExhaustVentilation(2);
-    constexpr int BalancedVentilation(3);
+    enum class Vent
```

Comment:
Can probably call this VentilationType and remove the word Ventilation from the enumerated values.